src/gstools/covmodel/fit.py

```diff
@@ -308,9 +309,11 @@ def _pre_init_guess(model, init_guess, mean_x=1.0, mean_y=1.0):
     for opt in model.opt_arg:
         init_guess.setdefault(
             opt,
-            default_arg_from_bounds(bnd[opt])
-            if default
-            else getattr(model, opt),
+            (
+                default_arg_from_bounds(bnd[opt])
+                if default
+                else getattr(model, opt)
+            ),
```

Comment:
This is the only black 24 change that is not related to the blank line below the module docstring.